### PR TITLE
OSDOCS-8693: add Getting Support assembly MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8693

Link to docs preview:
[Getting support](https://68476--docspreview.netlify.app/microshift/latest/microshift_support/microshift-getting-support)
[OCP Getting support](https://68476--docspreview.netlify.app/openshift-enterprise/latest/support/getting-support)

Support review:
- [x] Support has approved this change.

Additional information:
Also replaces DDF with module that links to creating a Jira

### CODE REVIEW

File: modules/support.adoc [hunk 1]
```diff
@@ -17,20 +17,21 @@
...
 
 [id="support_{context}"]
 = Getting support
 
 If you experience difficulty with a procedure described in this documentation, or with {product-title} in general, visit the link:http://access.redhat.com[Red Hat Customer Portal]. From the Customer Portal, you can:
 
 * Search or browse through the Red Hat Knowledgebase of articles and solutions relating to Red Hat products.
-* Submit a support case to Red Hat Support.
-// TODO: xref
+* Submit a support case to Red Hat Support
```
Comment (adellape) [inline modules/support.adoc:28]:
```suggestion
* Submit a support case to Red Hat Support.
```

File: modules/support.adoc [hunk 2]
```diff
@@ -17,20 +17,21 @@
 // * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-7.adoc
 // * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-8.adoc
 // * distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc
+// * microshift_support/microshift-getting-support.adoc
 
 [id="support_{context}"]
 = Getting support
 
 If you experience difficulty with a procedure described in this documentation, or with {product-title} in general, visit the link:http://access.redhat.com[Red Hat Customer Portal]. From the Customer Portal, you can:
```
Comment (adellape) [inline modules/support.adoc:25]: Maybe update to avoid continuing the sentence into the list.

**Comment by aldiazRH** [comment]
LGTM!

**Review by jldohmann — APPROVED** [review]
LGTM, nice work Shauna!

**Comment by jldohmann** [comment]
@ShaunaDiaz it looks like you'll have to manually CP to the 4.14 and 4.15 branches 🙃 feel free to ping me when those are up and i'll merge for you

**Comment by vwalek** [comment]
Hey, sorry, just read this sentence and do not understand what it means
~~~
Include data gathered using the sos tool or etcd as a starting point, 
plus any issue-specific data that is not collected in those logs.
~~~

There is no etcd tool, should be "inlcude data gathered using sos tool plus any issue-specific data."
I will report that as a new issue in jira.

**Comment by ShaunaDiaz** [comment]
> Hey, sorry, just read this sentence and do not understand what it means
> 
> ```
> Include data gathered using the sos tool or etcd as a starting point, 
> plus any issue-specific data that is not collected in those logs.
> ```
> 
> There is no etcd tool, should be "inlcude data gathered using sos tool plus any issue-specific data." I will report that as a new issue in jira.

See docs here: https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.14/html/support/microshift-etcd

**Comment by bergerhoffer** [comment]
@ShaunaDiaz @jldohmann It looks like this is failing because https://github.com/openshift/openshift-docs/pull/67345 only went into 4.15. This should have gone into main too

@ShaunaDiaz Any time you are touching ANY files outside of the release notes file, it has to go into main first. Can you please get the necessary changes from that PR into main?

It also looks like this PR needs to get into main as well: https://github.com/openshift/openshift-docs/pull/67051

I think you'll still need to do a manual cherrypick here though, because it's main where the items need to be resolved. But after all of this is resolved, then 4.15 and main should be in sync for these files currently in conflict.

The only file that should be out of sync is the actual release notes file.

**Comment by ShaunaDiaz** [comment]
> @ShaunaDiaz @jldohmann It looks like this is failing because #67345 only went into 4.15. This should have gone into main too
> 
> @ShaunaDiaz Any time you are touching ANY files outside of the release notes file, it has to go into main first. Can you please get the necessary changes from that PR into main?
> 
> It also looks like this PR needs to get into main as well: #67051
> 
> I think you'll still need to do a manual cherrypick here though, because it's main where the items need to be resolved. But after all of this is resolved, then 4.15 and main should be in sync for these files currently in conflict.
> 
> The only file that should be out of sync is the actual release notes file.

@bergerhoffer https://github.com/openshift/openshift-docs/pull/67345 is the placement of the release notes file in 4.15 and should not go in the main (this the actual adoc, not the build file that went in main)

https://github.com/openshift/openshift-docs/pull/67051 isn't my PR, I'm not certain what I can do with this one?

**Pull request merged**